components/odh-notebook-controller/main.go

```diff
@@ -128,6 +128,7 @@ func main() {
 		Log:       ctrl.Log.WithName("controllers").WithName("Notebook"),
```

Comment:
Wouldn't it make sense to create a new controller under this controller manager, that would only take care of creating a configmap for runtime images, in it's own reconcile loop?

Reply:
That you describe is a new implementation i guess

Reply:
No, I'm reading some kubebuilder docs and it got me thinking how this feature would've been implemented, following documented practices in https://book.kubebuilder.io/reference/good-practices#why-should-one-avoid-a-system-design-where-a-single-controller-is-responsible-for-managing-multiple-crds-custom-resource-definitionsfor-example-an-install_all_controllergo

Since the configmap creation is independent of anything else that happens in the reconcile loop, maybe it should have it's own reconcile loop.